@types/pal/audio.d.ts

```diff
@@ -0,0 +1,121 @@
+declare module 'pal/audio' {
+    export interface OneShotAudio {
```

Comment:
I can not give professional advice on interface design but code styles since lack of audio knowledges :). Could you please format the comments before this can be merged? ie. caption,  punctuations.

Reply:
Could you please give full description to `OneShotAudio` itself?

Reply:
done